src/main/resources/edu/cornell/kfs/rass/cu-spring-rass.xml

```diff
@@ -185,23 +185,23 @@
+				<bean parent="rassProperty" p:xmlPropertyName="status" p:boPropertyName="proposalStatusCode" p:required="false" p:valueConverter-ref="rassProposalStatusConverter" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="agencyNumber" p:boPropertyName="agencyNumber" p:required="true" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="projectTitle" p:boPropertyName="proposalProjectTitle" p:required="false" p:truncateWithEllipsis="true" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="startDate" p:boPropertyName="proposalBeginningDate" p:required="false" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="stopDate" p:boPropertyName="proposalEndingDate" p:required="false" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="directCostAmount" p:boPropertyName="proposalDirectCostAmount" p:required="true" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="indirectCostAmount" p:boPropertyName="proposalIndirectCostAmount" p:required="true" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="totalAmount" p:boPropertyName="proposalTotalProjectAmount" p:required="true" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="purpose" p:boPropertyName="proposalPurposeCode" p:required="false" p:valueConverter-ref="rassPurposeCodeConverter" p:skipForObjectEdit="true"/>
 				<bean parent="rassProperty" p:xmlPropertyName="grantNumber" p:boPropertyName="grantNumber" p:required="false"/>
 				<bean parent="rassProperty" p:xmlPropertyName="federalPassThrough" p:boPropertyName="proposalFederalPassThroughIndicator" p:required="false"
-						p:valueConverter-ref="rassPrimitiveBooleanConverter"/>
-				<bean parent="rassProperty" p:xmlPropertyName="federalPassThroughAgencyNumber" p:boPropertyName="federalPassThroughAgencyNumber" p:required="false"/>
+						p:valueConverter-ref="rassPrimitiveBooleanConverter" p:skipForObjectEdit="true"/>
+				<bean parent="rassProperty" p:xmlPropertyName="federalPassThroughAgencyNumber" p:boPropertyName="federalPassThroughAgencyNumber" p:required="false" p:skipForObjectEdit="true"/>
 				<bean parent="rassListPropertyWithProposalNumberMapping" p:xmlPropertyName="organizationCode" p:boPropertyName="proposalOrganizations" p:required="false"
 						p:valueConverter-ref="rassProposalOrganizationConverter" p:subObjectDefinition-ref="proposalOrganizationDefinition"/>
-				<bean parent="rassProperty" p:xmlPropertyName="finalReportDueDate" p:boPropertyName="proposalDueDate" p:required="false"/>
+				<bean parent="rassProperty" p:xmlPropertyName="finalReportDueDate" p:boPropertyName="proposalDueDate" p:required="false" p:skipForObjectEdit="true"/>
 				<bean parent="rassListPropertyWithProposalNumberMapping" p:xmlPropertyName="principalAndCoPrincipalInvestigators" p:boPropertyName="proposalProjectDirectors"
```

Comment:
this configuration looks like organizationCode and principalAndCoPrincipalInvestigators are editable fields, in addition to grantNumber.  I thought we just wanted grantNumber to be editable.  Would it make sense to have skipForObjectEdit defaulted to false, so only the fields the need to allow edits were need the new XML configuration

Reply:
"skipForObjectEdit" is already being defaulted to false, and it's only the Proposal setup that needs to set the value to true at this time. I'll see if I can create some separate parent beans that default "skipForObjectEdit" to true.

Also, as noted in the PR description and the JIRA comments, I've added a temporary fix to repopulate certain properties if the Award document somehow mistakenly cleared them out. (The org code and PI/director properties are the ones that need repopulating, and even then I've set up ProposalTranslationDefinition so that the updates won't happen unless the Grant Number has changed.) Without this fix, most Grant Number updates to Proposals will run into business rule failures. (Our legacy Ezra integration updated the Grant Number via the BO service to get around this.)